google-cloud-storage/src/test/java/com/google/cloud/storage/ITHttpOpenTelemetryTest.java

```diff
@@ -97,6 +109,62 @@ public void runCreateBlob() {
     }
   }
 
+  @Test
+  public void runRead() throws IOException {
+    BlobInfo blobInfo =
+        BlobInfo.newBuilder(blobId).setContentEncoding("gzip").setContentType("text/plain").build();
+    storage.create(blobInfo, helloWorldGzipBytes);
+    Path helloWorldTxtGz = File.createTempFile(blobId.getName(), ".txt.gz").toPath();
+    storage.downloadTo(
+        blobId, helloWorldTxtGz, Storage.BlobSourceOption.shouldReturnRawInputStream(true));
+    TestExporter testExported = (TestExporter) exporter;
+    List<SpanData> spanData = testExported.getExportedSpans();
+    for (SpanData span : spanData) {
```

Comment:
`spanData` can be de-duped as a helper that validates attributes

Reply:
what do you mean by this?

Reply:
```
List<SpanData> spanData = testExported.getExportedSpans();
    for (SpanData span : spanData) {
      Assert.assertEquals("Storage", getAttributeValue(span, "gcp.client.service"));
      Assert.assertEquals("googleapis/java-storage", getAttributeValue(span, "gcp.client.repo"));
      Assert.assertEquals(
          "com.google.cloud.google-cloud-storage", getAttributeValue(span, "gcp.client.artifact"));
      Assert.assertEquals("http", getAttributeValue(span, "rpc.system"));
    }
    ```

Is duplicated for runRead() and runCopy(); thinking a helper method can be used to reduce duplication of the same asserts.